app/src/main/java/com/eveningoutpost/dexdrip/WidgetUpdateService.java

```diff
@@ -55,61 +35,41 @@ public WidgetUpdateService() {}
 
     @Override
     public void onCreate() {
+        context = getApplicationContext();
```

Comment:
Don't store the application context in a static field here, just use `xdrip.getAppContext()` if you need it from static, otherwise just store it in a local variable.

Reply:
I'll make the change.